css/main.css

```diff
@@ -1269,7 +1271,7 @@ html:not(.has-config) #status-bar .configuration-file-name {
 
 ***/
 
-input[type="checkbox"] { 
+input[type="checkbox"].ios-switch { 
```

Comment:
What's the `.ios-switch` for?

Reply:
All checkboxes are rendered as ios-switches, but in this table I had too limited space. So I wanted a checkbox instead. Also an switch was a bit confusing. "Am I turning something on? Why is my field turned off by default?"

Reply:
Adding screenshots of proposed changes to the pull request often helps with illustrating what the change does.

Reply:
![image](https://user-images.githubusercontent.com/5307773/69916024-e978b880-1455-11ea-954e-b207865b4101.png)

Before this change any checkbox without ios-switch class was hidden. The ios-switch hides the checkbox and amend it's own graphics.